avalon/photoshop/lib.py

```diff
@@ -0,0 +1,255 @@
+import json
+import contextlib
+import functools
+
+from win32com.client import Dispatch
+
+from . import com_objects
+from ..tools import html_server
+
+# Convinience variable that could later query whether the platform is Windows
+# or Mac.
+# This needs to be a partial function we can later call because when calling
+# Dispatch directly from a different thread will result first a
+# "CoInitialize has not been called.", which can be fixed with
+# pythoncom.CoInitialize() but secondly "The application called an interface
+# that was marshalled for a different thread".
```

Comment:
This comment seems a bit oddly phrased.

> This needs a partial function call because when calling Dispatch directly from a different thread will result in "CoInitialize has not been called" which can be fixed with pythoncom.CoInitialize(). However even then it will still error with "The application called an interface that was marshalled for a different thread"

Does that make it more comprehensible?

Reply:
Yeah, makes more sense. Thanks!